scapy/contrib/nlm.py

```diff
@@ -0,0 +1,307 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Lucas Preston <lucas.preston@infinite.io>
+# This program is published under a GPLv2 license
+
+# scapy.contrib.description = Network Lock Manager (NLM) v4
+# scapy.contrib.status = loads
+
+from scapy.contrib.oncrpc import RPC, RPC_Call
+from binascii import unhexlify
+from scapy.packet import Packet, bind_layers
+from scapy.fields import IntField, StrLenField, LongField, PacketField, \
+    IntEnumField
+
+nlm4_stats = {
+    0: 'NLM4_GRANTED',
+    1: 'NLM4_DENIED',
+    2: 'NLM4_DENIED_NOLOCKS',
+    3: 'NLM4_BLOCKED',
+    4: 'NLM4_DENIED_GRACE_PERIOD',
+    5: 'NLM4_DEADLCK',
+    6: 'NLM4_ROFS',
+    7: 'NLM4_STALE_FH',
+    8: 'NLM4_FBIG',
+    9: 'NLM4_FAILED'
+}
+
+
+class File_Object(Packet):
```

Comment:
Can't you simply do
`from scapy.contrib.mount import File_Object`

Reply:
This and more changes along these lines are done in `48dc21e4`